ts/editor/image-overlay/ImageHandle.svelte

```diff
@@ -152,12 +152,13 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         }
 
         /**
-         * Image resizing add-ons previously used image.style.width to set the
+         * Image resizing add-ons previously used image.style.width/height to set the
          * preferred width of an image. In these cases, if we'd only set
          * image.width, there would be no visible effect on the image.
          * To avoid confusion with users we'll clear image.style.width (for now).
```

Comment:
The comment is a bit confusing now. Maybe reword it somewhat like this?:

* Image resizing add-ons previously used image.style.width/height to set the
* preferred dimensions of an image. In these cases, if we'd only set
* image.[dimension], there would be no visible effect on the image.
* To avoid confusion with users we'll clear image.style.[dimension] (for now).

Open for suggestions.

Reply:
I didn't read the next lines. Fixed!